test/e2e/socialevent_test.go

```diff
@@ -0,0 +1,72 @@
+package e2e_test
+
+import (
+	"context"
+	"testing"
+
+	toolchainv1alpha1 "github.com/codeready-toolchain/api/api/v1alpha1"
+	"github.com/codeready-toolchain/toolchain-e2e/testsupport"
+	. "github.com/codeready-toolchain/toolchain-e2e/testsupport"
+	. "github.com/codeready-toolchain/toolchain-e2e/testsupport/wait"
+
+	"github.com/stretchr/testify/require"
+	corev1 "k8s.io/api/core/v1"
+)
+
+func TestCreateSocialEvent(t *testing.T) {
+	// given
+
+	// make sure everything is ready before running the actual tests
+	awaitilities := WaitForDeployments(t)
+	hostAwait := awaitilities.Host()
+
+	t.Run("create socialevent with valid tiername", func(t *testing.T) {
+		// given
+		se := testsupport.NewSocialEvent("lab-", "base")
+
+		// when
+		err := hostAwait.CreateWithCleanup(context.TODO(), se)
+
+		// then
+		require.NoError(t, err)
+		_, err = hostAwait.WaitForSocialEvent("lab", UntilSocialEventHasConditions(toolchainv1alpha1.Condition{
```

Comment:
correct me if I'm wrong, but when creating the SocialEvent CR you used the `lab-` with the dash suffix, which would mean that it should use the generated name, so you cannot pass the `lab` name here, because it would be different.
BTW, couldn't we just use generated names by default and pass them via `se.Name`?